goi18n/merge_test.go

```diff
@@ -10,7 +10,7 @@ import (
 func TestMergeExecute(t *testing.T) {
 	resetDir(t, "testdata/output")
 	files := []string{
-		"testdata/input/en-us.one.json",
+		"testdata/input/en-us.one.yaml",
```

Comment:
Instead of editing this test, can you leave this test as-is and add a new test which uses the new yaml file? Basically copy/paste.